client/landing/stepper/declarative-flow/free.ts

```diff
@@ -0,0 +1,140 @@
+import { useLocale } from '@automattic/i18n-utils';
+import { useFlowProgress, FREE_FLOW } from '@automattic/onboarding';
+import { useSelect, useDispatch } from '@wordpress/data';
+import { useEffect } from 'react';
+import { recordFullStoryEvent } from 'calypso/lib/analytics/fullstory';
+import { recordTracksEvent } from 'calypso/lib/analytics/tracks';
+import wpcom from 'calypso/lib/wp';
+import {
+	clearSignupDestinationCookie,
+	setSignupCompleteSlug,
+	persistSignupDestination,
+	setSignupCompleteFlowName,
+} from 'calypso/signup/storageUtils';
+import { useSiteSlug } from '../hooks/use-site-slug';
+import { USER_STORE, ONBOARD_STORE } from '../stores';
+import { recordSubmitStep } from './internals/analytics/record-submit-step';
+import type { StepPath } from './internals/steps-repository';
+import type { Flow, ProvidedDependencies } from './internals/types';
+
+export const free: Flow = {
+	name: FREE_FLOW,
+	title: 'Free',
+	useSteps() {
+		useEffect( () => {
+			recordTracksEvent( 'calypso_signup_start', { flow: this.name } );
+			recordFullStoryEvent( 'calypso_signup_start_free', { flow: this.name } );
+		}, [] );
+
+		// return [ 'intro', 'freeSetup', 'launchpad' ] as StepPath[];
+		return [
+			'intro',
+			'linkInBioSetup',
+			'domains',
```

Comment:
`NB`: We don't want domains and plans for this flow

Reply:
Yup, there's some complexity to removing them. I figured we'd handle each of those separately. I've added tasks for those to the Free Flow tracking issue.